settings.py

```diff
@@ -8,7 +12,11 @@
 ]
 
 import aldryn_addons.settings
-aldryn_addons.settings.load(locals())
+
+aldryn_addons.settings.load(
```

Comment:
I'd like to keep the footprint of the call to ``aldryn_addons.settings.load()`` as minimal as possible. I'd prefer the ``debug`` attribute not to be necessary.